### PR TITLE
Refactored code and installation page

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**

### CODE REVIEW

File: component/Cards/styles.css [hunk 1]
```diff
@@ -41,18 +34,14 @@
 .card-content-txt-color {
     color: gray;
```
Comment (varun1233) [inline component/Cards/styles.css]: use variables, as you are using variables for other colors.

File: component/Grids/index.html [hunk 2]
```diff
@@ -79,7 +81,7 @@ <h4 class="component-subheading"> Grid with two column </h4>
 
         <!-- GRID 3 COLUMN -->
 
-        <h4 class="component-subheading"> Grid with three column </h4>
+        <h4 class="component-subheading">> Grid with three column </h4>
```
Comment (varun1233) [inline component/Grids/index.html:84]: Remove extra closing bracket ">"

Comment (vanshitamoolchandani) [reply component/Grids/index.html:84]: Its just for showing sub points.
image

File: component/Cards/styles.css (hunk 1 above)
Comment (vanshitamoolchandani) [reply component/Cards/styles.css]: done

**Review by varun1233 — APPROVED** [review]

File: component/Buttons/styles.css [hunk 3]
```diff
@@ -17,7 +17,9 @@
     border-radius: 0.5rem;
```
Comment (yaswanthmyneni) [inline component/Buttons/styles.css:17]: changes: use px units for values less than 1rem.

File: component/Cards/styles.css [hunk 4]
```diff
@@ -38,21 +31,15 @@
     font-size: var(--content-size);
     font-weight: var(--content-weight);
 }
-.card-content-txt-color {
-    color: gray;
-}
 
 .card-txt-red {
     text-decoration:line-through;
     color: red;
```
Comment (yaswanthmyneni) [inline component/Cards/styles.css:37]: changes: use css variables for color.

File: component/Cards/styles.css [hunk 5]
```diff
@@ -65,18 +52,13 @@
     position: relative;
     z-index: 1;
```
Comment (yaswanthmyneni) [inline component/Cards/styles.css:53]: suggestion: can use css variables for z-indexs.

File: component/NavBar-Vertical/styles.css [hunk 6]
```diff
@@ -34,20 +32,17 @@
     width: 200px;
```
Comment (yaswanthmyneni) [inline component/NavBar-Vertical/styles.css:32]: changes: use rem for units more than 16px.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after chnages.

**Review by Sahil25061999 — APPROVED** [review]
code mostly looks good, Ui is also good, just a suggestion the orange text on blue background for the avatar section doesnt have good contrast ratio so i think that can be changed as per your liking. Otherwise it is good to merge

File: component/Cards/styles.css (hunk 4 above)
Comment (vanshitamoolchandani) [reply component/Cards/styles.css:37]: Done

File: component/Cards/styles.css (hunk 5 above)
Comment (vanshitamoolchandani) [reply component/Cards/styles.css:53]: Thank you. Will keep in mind

File: component/NavBar-Vertical/styles.css (hunk 6 above)
Comment (vanshitamoolchandani) [reply component/NavBar-Vertical/styles.css:32]: Done

**Pull request merged**